### ISSUE TITLE
Same default answer for routing

### ISSUE THREAD

**Issue by daniloesk** [single-spa/single-spa-angular#310]
### Description of Proposed Feature

Not really a feature, more like an improvement.
During create-single-spa of an Angular project, we have two questions about routing with different default options.

The first one from Angular, I suppose:
```
? Would you like to add Angular routing? (y/N)
```

And the next from single-spa-angular, guessing again:
```
? Does your application use Angular routing? (Y/n)
```

Probably it would be better to have the same default option to improve user experience and reduce errors.
Of course, removing the second answer would be perfect, but I guess a simpler tweak already helps a lot.

### In what way would you use it?
Help me avoid creating misconfigured Angular projects.

**Comment by joeldenning** [comment]
Agreed that this would be nice - i'm not sure how to implement it. If anyone would like to create a pull request for it, I'm happy to help review it.

**Comment by arturovt** [comment]
This is not possible due to some technical reasons.

This question presents because not all people use _create-single-spa_, but some may add this package via _ng add single-spa-angular_ into some existing project.

Since we can't determine if the user uses the routing in his application we can't skip that question. Checking that `app-routing` file exists or `AppRoutingModule` is imported into an `AppModule` is a wrong way since those files/classes can be named anything.

We could set default value to `true`. But I am afraid that no one will intentionally press `Enter` and routing will be added to the application where it should not be.

**Comment by daniloesk** [comment]
@arturovt, I realized the way using `ng add` later and it makes total sense. Actually I stopped using create-single-spa at all, because it does not have all the options required to create a proper Angular project for single-spa (e.g. prefix).

Good point on minimal as default and usually I would like that option myself. But `ng new` does use routing by default so the idea would be having the same behavior. Exactly in the case  `Enter` was used without thinking, we would have a routing Angular and no routing single-spa. Of couse it would be all the user's fault, since we did present him with the option, but again, this is a suggestion.

Although not using `AppRoutingModule` name is possible how many projects actually change that? Other option is looking for occurance of RouterModule inside of src. That can be set just as the default, but that may be more confusing.

This seems to be getting to complicated to little gain. =/
Probably just leave it to the future if needed.
